src/librawspeed/decoders/ArwDecoder.cpp

```diff
@@ -118,8 +119,9 @@ RawImage ArwDecoder::decodeRawInternal() {
       // And now decode as a normal 16bit raw
       mRaw->dim = iPoint2D(width, height);
       mRaw->createData();
-      ByteStream input(mFile, off, len);
-      Decode16BitRawBEunpacked(input, width, height);
+
+      UncompressedDecompressor u(*mFile, off, len, mRaw, uncorrectedRawValues);
+      u.Decode16BitRawBEunpacked(width, height);
```

Comment:
> Also, simply because I personally think UncompressedDecompressor sounds bit 'contradicting': maybe something like PlainDecompressor or UnpackDecompressor would fit better?

I don't think so. [Dng Spec](http://wwwimages.adobe.com/content/dam/Adobe/en/products/photoshop/pdfs/dng_spec_1.4.0.0.pdf) calls it: `1: Uncompressed data.`. So it is it's name.